tools/mirror/src/cli.rs

```diff
@@ -67,16 +61,17 @@ impl RunCmd {
             }
             None
         };
-
+        
         let system = new_actix_system(runtime);
-        system
-            .block_on(async move {
-                let _subscriber_guard = near_o11y::default_subscriber(
-                    near_o11y::EnvFilterBuilder::from_env().finish().unwrap(),
-                    &near_o11y::Options::default(),
-                )
-                .global();
-                actix::spawn(crate::run(
+        system.block_on(async move {
+            let _subscriber_guard = near_o11y::default_subscriber(
+                near_o11y::EnvFilterBuilder::from_env().finish().unwrap(),
+                &near_o11y::Options::default(),
+            )
+            .global();
+            
+            /*
```

Comment:
We generally strive to not have commented-out code… You probably should remove this.